src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2670,6 +2670,7 @@ private function registerMailerConfiguration(array $config, ContainerBuilder $co
         }
 
         $classToServices = [
+            MailerBridge\AhaSend\Transport\AhaSendTransportFactory::class => 'mailer.transport_factory.ahasend',
```

Comment:
```suggestion
            MailerBridge\AhaSend\Transport\AhaSendTransportFactory::class => 'mailer.transport_factory.aha_send',
```
Do you agree @fabpot ?